### PR TITLE
apply docs recommendation

### PR DESCRIPTION
**Pull request by erwinmombay**
Closes https://github.com/ampproject/amphtml/issues/3468

### CODE REVIEW

**Comment by erwinmombay** [comment]
@ericlindley-g PTAL

File: extensions/amp-live-list/amp-live-list.md [hunk 1]
```diff
@@ -247,8 +247,10 @@ seconds that have elapsed since Thursday, 1 January 1970).
 
 **data-update-time** (Optional)
 
-Timestamp when the entry was last updated. We recommend using Unix time
-(the number of seconds that have elapsed since Thursday, 1 January 1970).
+Timestamp when the entry was last updated.  Use this attribute to trigger an
+update on an existing item, the client will do an outright replace of the
```
Comment (ericlindley-g) [inline extensions/amp-live-list/amp-live-list.md]: I'm not sure readers will understand what an "outright replace" means.

How about:

"Use this attribute to trigger an update on an existing item: the client will replace all existing content in this item with the  new, updated content, without triggering the appearance of the `update` reference point"

Does this get at what we want to say here?

Comment (erwinmombay) [inline extensions/amp-live-list/amp-live-list.md]: done.

**Comment by erwinmombay** [comment]
@ericlindley-g PTAL

**Comment by ericlindley-g** [comment]
Thanks—LGTM

**Pull request merged**